src/gui/spaces/CMakeLists.txt

```diff
@@ -1,25 +1,7 @@
-add_library(ownCloudSpaces SHARED
+target_sources(owncloudGui
```

Comment:
I finally had to merge the spaces browser into the gui module....
In order to track which spaces are synced I needed to access the folder manager.....